i18n/en.json

```
@@ -440,9 +439,11 @@
   "admin.environment.smtp.smtpServer.description": "Location of SMTP email server.",
   "admin.environment.smtp.smtpServer.placeholder": "Ex: \"smtp.yourcompany.com\", \"email-smtp.us-east-1.amazonaws.com\"",
   "admin.environment.smtp.smtpServer.title": "SMTP Server:",
+  "admin.environment.smtp.smtpSuccess": "No errors were reported while sending an email.  Please check your inbox to make sure.",
```

nit: looks like we use one space after periods in most messages
```suggestion
  "admin.environment.smtp.smtpSuccess": "No errors were reported while sending an email. Please check your inbox to make sure.",
```

---

Fixed :)